mkdocs.yml

```diff
@@ -26,8 +26,8 @@ theme:
 
 extra:
   analytics:
-    provider: google
-    property: G-E31MPDGETR
+    provider: segment
+    property: "4sT1ml0BeKdR1RtrK5dSQmwxmvcUpYtL"
```

Comment:
yes, we should use a different property, otherwise, we'll load all the integrations that Codacy is using
with a different property we can choose which tools to loads on each product - even if we end up with the same set of tools, it might be worth having the choice